fold-and-r.csl

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US">
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US">
```

Comment:
you also need to change line 12 below https://github.com/citation-style-language/styles/pull/1464/files#diff-a03802aca1da523688424503e1158fcfR12 to note instead of author-date. That's why the automated test fails.